tools/tests/apps/compiler-plugin-asset-and-source/.meteor/.finished-upgraders

```diff
@@ -0,0 +1,8 @@
+# This file contains information which helps Meteor properly upgrade your
```

Comment:
I think it's OK to leave `.finished-upgraders`, `.id`, `platforms` (if the default), and `versions` out of test apps. It's not like any process is going to update finished-upgraders or versions in git when the versions change...